requirements.txt

```diff
@@ -1,4 +1,4 @@
-pvlib>=0.6.0,<0.8.0
+pvlib>=0.7.0,<0.9.0
```

Comment:
Thanks @kanderso-nrel !
In my head, the upper bound I put in in the latest patch ("<0.8.0") was kind of temporary until I would find time to implement the fixes and bump the pvlib version. My original philosophy was to always try to use the latest version of pvlib, and let new versions break the tests so that I would know if there's an update needed in pvfactors.
But since I have very limited time for this project lately and not many contributions or messages (thanks again for yours!), I think that for now we can keep it, and if it causes any issue we will hear about it very soon.
👍